### ISSUE TITLE
Could `pred_fake = netD_B(fake_B.detach())` be negative

### ISSUE THREAD

**Issue by TheLostIn** [huawei-noah/GAN-pruning#2]
Hello,I was running the `search.py`,and when I run it on Win10,the value `pred_fake = netD_B(fake_B.detach())` was positive and it was negative on ubuntu.
Could `pred_fake = netD_B(fake_B.detach())` be negative,is it normal?

**Comment by shuh15** [comment]
I think it is normal, the discriminator net doesn't scale the output to be positive.
